h2o-r/h2o-package/R/connection.R

```diff
@@ -492,13 +492,12 @@ h2o.clusterStatus <- function() {
     "----------------------------------------------------------------------\n",
     "\n",
     "Your next step is to start H2O:\n",
```

Comment:
Now that I read this again, I think we should remove this whole block. Having this print out just
`Your next step is to start H2O: >` seems not very user friendly.
I would either remove this all together or replace with
`You next step is to connect to your H2O cluster: > h2o.connect(url="http://hostname:port")`

Reply:
I see my mistake, this will be replaced with h2o.connect, so this is fine.